catroid/res/values-ar/strings.xml

```diff
@@ -391,11 +391,11 @@
   <string name="stage_dialog_maximize">زيادة الامتداد\n</string>
   <string name="stage_dialog_screenshot">لقطة للشاشة</string>
   <!-- Broadcast spinner -->
-  <string name="new_broadcast_message">جديد&#8230;</string>
+  <string name="new_broadcast_message">جديد…</string>
```

Comment:
Sure! I'll have a look at it :wink, as soon as I find the time. Thank you for your input.